community-mesh-vpn-tunneldigger/Makefile

```diff
@@ -0,0 +1,16 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=community-mesh-vpn-tunneldigger
+PKG_VERSION:=0.0.1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0-or-later
```

Comment:
Would be good if someone could verify this is the correct license.

Reply:
Gluon itself is BSD C-3:

https://github.com/freifunk-gluon/gluon/blob/master/LICENSE

Reply:
actually i think it's [BSD 2-Clause](https://opensource.org/license/bsd-2-clause/), i'll update it

Reply:
Thanks..
I created a PR for Gluon so that this does not spread confusion in the future:
https://github.com/freifunk-gluon/gluon/pull/3112